comp/logs/streamlogs/impl/streamlogs.go

```diff
@@ -70,11 +125,11 @@ func (sl *streamlogsimpl) exportStreamLogsIfEnabled(logAgent logsAgent.Component
 
 	if values, ok := enableStreamLog.([]interface{}); ok && len(values) > 1 {
 		if enable, ok := values[1].(bool); ok && enable {
-			streamLogParams := stream.LogParams{
+			streamLogParams := LogParams{
 				FilePath: streamlogsLogFilePath,
 				Duration: 60 * time.Second, // Default duration is 60 seconds
 			}
-			if err := stream.ExportStreamLogs(logAgent, &streamLogParams); err != nil {
+			if err := exportStreamLogs(logsAgent, &streamLogParams); err != nil {
```

Comment:
We can add this function as part of the `streamlogsimpl`

`sl.exportStreamLogs(&streamLogParams)` no need to pass the `logsAgent` around 😄